pkg/operator/stringmaps/stringmaps.go

```diff
@@ -0,0 +1,25 @@
+package stringmaps
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"sort"
+)
+
+func HashKeys(m map[string]string) string {
```

Comment:
Add doc comments on all exported functions:

```go
// HashKeys returns a hex-encoded hash of all the keys in the map.
//
// This can be used to compare two unordered sets of keys for equality.
// The keys are assumed to not contain newlines.
// The values in the map are ignored.
```

Reply:
Done.